.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Gridcoin Discord
+    url: https://discord.gg/jf9XX4a
+    about: Please go here if you have any general issues that aren't bug reports. We can assist you much faster there.
```

Comment:
I didn't see this before. No contractions in formal technical writing. aren't should be are not.